src/librustc_back/target/apple_ios_base.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2014 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use std::io::Command;
+use target::TargetOptions;
+
+pub fn get_sdk_root(sdk_name: &str) -> String {
+    let mut child = match Command::new("xcrun")
+                                  .arg("--show-sdk-path")
+                                  .arg("-sdk")
+                                  .arg(sdk_name)
+                                  .spawn() {
+        Ok(child) => child,
```

Comment:
You may want to also assert that the process exited with a 0 exit code as it may help debug when things go wrong.